src/angular/form-field/input.ts

```diff
@@ -245,8 +253,35 @@ export class SbbInput
     // If no input value accessor was explicitly specified, use the element as the input value
     // accessor.
     this._inputValueAccessor = inputValueAccessor || element;
+
     this._previousNativeValue = this.value;
 
+    // On some versions of iOS the caret gets stuck in the wrong place when holding down the delete
+    // key. In order to get around this we need to "jiggle" the caret loose. Since this bug only
+    // exists on iOS, we only bother to install the listener on iOS.
+    if (_platform.IOS) {
```

Comment:
I'm not sure whether this is needed for our use cases, but it also shouldn't hurt